### ISSUE TITLE
How to get the list of resolution qualities?

### ISSUE THREAD

**Issue by vanminhquangtri** [shaka-project/shaka-player#5483]
Hello Brothers,

I am using Shaka player 4.0.2 to play DRM content. My App is Vue 2 + Nuxt 2 on Microsoft Edge.

I wonder if Shaka player has any built-in function/method to get the list of resolution qualities of the current playing content (I mean Shaka can auto detect the list of resolution quality: for example Auto adaptive, 480p, 720p, 1080p,...).

We only have 1 stream link for this content which look like:
https://example.cdn.net/live_dzones/max.smil/manifest.mpd.

I have searched in Internet as well as your docs but I found nothing.

Thank you.

**Comment by WINOFFRG** [comment]
Hi! It's here `getVariantTracks` https://shaka-player-demo.appspot.com/docs/api/shaka.Player.html#getVariantTracks It should be able to give you the resolution qualities.

**Comment by vanminhquangtri** [comment]
@WINOFFRG thanks for your rep. I see it return list of shaka.extern.Track
I am confused that this case "track" mean text track/ audio track? Am I wrong?

**Comment by WINOFFRG** [comment]
As the method name `getVariantTracks` you will get list of all combinations of A/V. So if a stream has 3 Video Tracks and 2 Audio Tracks this will return you 6 combinations. To get the language and resolution or meta for that specific track just look at the defintion of `shaka.extern.Track`.

TL;DR It return the combination of both.
